### PR TITLE
Added property to Icepak Edit Design Settings

### LINKED ISSUE #3612: Add gauge pressure and radiation temp to Icepak.edit_design_settings()
### Description of the feature

Currently, the function Icepak.edit_design_settings() can't set gauge pressure and radiation temperature. They are in the first tab of the "Icepak Design Settings" dialog in AEDT GUI.
AEDT_design_settings

### Steps for implementing the feature

_No response_

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by maxcapodi78**
Fix #3612

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -1475,7 +1494,7 @@ def edit_design_settings(
             [
                 "NAME:Model Validation Settings",
                 "EntityCheckLevel:=",
-                CheckLevel,
+                check_level,
                 "IgnoreUnclassifiedObjects:=",
                 False,
                 "SkipIntersectionChecks:=",
```
[the remark this answers is not in the document]
Comment (maxcapodi78) [reply pyaedt/icepak.py:1507]: @nnetake I've added both ignore_unclassified_objects and skip_intersection_checks

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -1433,17 +1451,27 @@ def edit_design_settings(
...
         GVPos = False
-        if int(gravityDir) > 2:
+        if "gravityDir" in kwargs:
+            warnings.warn(
+                "`gravityDir` is deprecated. Use `gravity_dir` instead.",
+                DeprecationWarning,
+            )
+
+            gravity_dir = kwargs["gravityDir"]
+        if "CheckLevel" in kwargs:
```
Comment (Samuelopez-ansys) [inline pyaedt/icepak.py:1465]:
```suggestion
        if "CheckLevel" in kwargs:  # pragma: no cover
```

File: pyaedt/icepak.py [hunk 3]
```diff
@@ -1433,17 +1451,27 @@ def edit_design_settings(
...
-        if ambtemp and not isinstance(ambtemp, str):
-            AmbientTemp = str(ambtemp) + "cel"
-        else:
-            AmbientTemp = ambtemp
         #
         # Configure design settings for gravity etc
         IceGravity = ["X", "Y", "Z"]
         GVPos = False
-        if int(gravityDir) > 2:
+        if "gravityDir" in kwargs:
```
Comment (Samuelopez-ansys) [inline pyaedt/icepak.py:1458]:
```suggestion
        if "gravityDir" in kwargs:  # pragma: no cover
```

File: _unittest/test_98_Icepak.py [hunk 4]
```diff
@@ -251,7 +251,7 @@ def test_13b_assign_grille(self):
         assert grille2.update()
 
     def test_14_edit_design_settings(self):
-        assert self.aedtapp.edit_design_settings(gravityDir=1)
+        assert self.aedtapp.edit_design_settings(gravity_dir=1)
```
Comment (Samuelopez-ansys) [inline _unittest/test_98_Icepak.py:254]:
```suggestion
        assert self.aedtapp.edit_design_settings(gravity_dir=1)
        assert self.aedtapp.edit_design_settings(gravity_dir=3)
```

**Pull request merged**